api/provider/aws/helpers.go

```diff
@@ -273,11 +273,16 @@ func (p *AWSProvider) stackUpdate(name string, templateUrl string, changes map[s
 
 		fp, err := formationParameters(string(body))
 
-		// remove params that don't exist in the template
 		for key := range params {
+			// remove params that don't exist in the template
 			if _, ok := fp[key]; !ok {
 				delete(params, key)
 			}
+
+			// exclude Password from params to avoid updating password to "*****"
```

Comment:
Is there some easy way to catch this higher and eliminate all `NoEcho` parameters from being carried forward into this map in the first place?